### PR TITLE
K8SPG-66 Percona PostgreSQL operator chart

### PR DESCRIPTION
**Pull request by cap1984**

### CODE REVIEW

File: charts/pg-operator/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+name: pg-operator
+description: 'A Helm chart for Deploying the Percona Kubernetes Operator for Percona PostgreSQL'
+type: application
+version: 0.2.0
+appVersion: 4.7.0
```
Comment (hors) [inline charts/pg-operator/Chart.yaml:6]: we do not have 4.7.0

File: charts/pg-operator/values.yaml [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+backrest_aws_s3_region: ""
+backrest_aws_s3_secret: ""
+backrest_aws_s3_uri_style: ""
+backrest_aws_s3_verify_tls: "true"
+backrest_gcs_bucket: ""
+backrest_gcs_endpoint: ""
+backrest_gcs_key_type: ""
+backrest_port: "2022"
+badger: "false"
+ccp_image_prefix: "perconalab/percona-postgresql-operator"
```
Comment (hors) [inline charts/pg-operator/values.yaml:35]:
```suggestion
ccp_image_prefix: "percona/percona-postgresql-operator"
```

File: charts/pg-operator/values.yaml [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+ccp_image_pull_secret: ""
+ccp_image_pull_secret_manifest: ""
+ccp_image_tag: "main-postgres-ha"
```
Comment (hors) [inline charts/pg-operator/values.yaml:38]: I think we need to add chart for 0.1.0 for now

File: charts/pg-operator/values.yaml [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+pgo_apiserver_port: "8443"
+pgo_apiserver_url: "https://postgres-operator"
+pgo_client_cert_secret: "pgo.tls"
+pgo_client_container_install: "false"
+pgo_client_install: "true"
+pgo_client_version: "4.7.0"
+pgo_cluster_admin: "false"
+pgo_disable_eventing: "false"
+pgo_disable_tls: "false"
+pgo_image_prefix: "perconalab/percona-postgresql-operator"
```
Comment (hors) [inline charts/pg-operator/values.yaml:76]:
```suggestion
pgo_image_prefix: "percona/percona-postgresql-operator"
```

**Review by hors — CHANGES_REQUESTED** [review]

File: charts/pg-operator/Chart.yaml (hunk 1 above)
Comment (cap1984) [reply charts/pg-operator/Chart.yaml]: done

File: charts/pg-operator/values.yaml (hunk 2 above)
Comment (cap1984) [reply charts/pg-operator/values.yaml]: done

File: charts/pg-operator/values.yaml (hunk 3 above)
Comment (cap1984) [reply charts/pg-operator/values.yaml]: done

File: charts/pg-operator/values.yaml (hunk 4 above)
Comment (cap1984) [reply charts/pg-operator/values.yaml]: done

File: charts/pg-operator/README.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+| Name | Default | Description |
+| ---- | ------- | ----------- |
+| fullnameOverride | "" |  |
+| rbac.create | true | If false RBAC will not be created. RBAC resources will need to be created manually and bound to `serviceAccount.name` |
+| rbac.useClusterAdmin | false | If enabled the ServiceAccount will be given cluster-admin privileges. |
+| serviceAccount.create | true | If false a ServiceAccount will not be created. A ServiceAccount must be created manually. |
+| serviceAccount.name | "" | Use to override the default ServiceAccount name. If serviceAccount.create is false this ServiceAccount will be used. |
+
+{{% notice tip %}}
+If installing into an OpenShift 3.11 or Kubernetes 1.11 cluster `rbac.useClusterAdmin` must be enabled.
```
Comment (tplavcic) [inline charts/pg-operator/README.md:43]: Probably we do not support OpenShift 3.11 and Kubernetes 1.11 - `## Pre-requisites` mentions kubernetes 1.16+

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: charts/pg-operator/README.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+## Chart Details
+This chart will:
+* deploy a PG Operator Pod for the further PostgreSQL creation in K8S.
+
+### Installing the Chart
+To install the chart with the `postgres-operator` release name using a dedicated namespace (recommended):
+
+```sh
+helm repo add percona https://percona.github.io/percona-helm-charts/
+helm install my-operator percona/postgres-operator --version 0.2.0 --namespace my-namespace
```
Comment (hors) [inline charts/pg-operator/README.md:26]:
```suggestion
helm install my-operator percona/postgres-operator --version 0.1.0 --namespace my-namespace
```

File: charts/pg-operator/values.yaml [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+pgo_admin_password: "examplepassword"
+pgo_admin_perms: "*"
+pgo_admin_role_name: "pgoadmin"
+pgo_admin_username: "admin"
+pgo_apiserver_port: "8443"
+pgo_apiserver_url: "https://postgres-operator"
+pgo_client_cert_secret: "pgo.tls"
+pgo_client_container_install: "false"
+pgo_client_install: "true"
+pgo_client_version: "4.7.0"
```
Comment (hors) [inline charts/pg-operator/values.yaml:72]:
```suggestion
pgo_client_version: "4.6.2"
```
or maybe we can remove it at all

File: charts/pg-operator/values.yaml [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+pgo_add_os_ca_store: "false"
+pgo_admin_password: "examplepassword"
+pgo_admin_perms: "*"
+pgo_admin_role_name: "pgoadmin"
+pgo_admin_username: "admin"
+pgo_apiserver_port: "8443"
+pgo_apiserver_url: "https://postgres-operator"
+pgo_client_cert_secret: "pgo.tls"
+pgo_client_container_install: "false"
+pgo_client_install: "true"
```
Comment (hors) [inline charts/pg-operator/values.yaml:71]:
```suggestion
pgo_client_install: "false"
```

**Review by hors — CHANGES_REQUESTED** [review]

File: charts/pg-operator/values.yaml (hunk 8 above)
Comment (cap1984) [reply charts/pg-operator/values.yaml]: done

File: charts/pg-operator/values.yaml (hunk 7 above)
Comment (cap1984) [reply charts/pg-operator/values.yaml]: done

File: charts/pg-operator/README.md (hunk 6 above)
Comment (cap1984) [reply charts/pg-operator/README.md]: done

File: charts/pg-operator/README.md (hunk 5 above)
Comment (cap1984) [reply charts/pg-operator/README.md]: done

**Review by tplavcic — APPROVED** [review]

**Review by hors — APPROVED** [review]

**Pull request merged**